bin/composite_rsr_plot.py

```diff
@@ -103,6 +103,11 @@ def get_arguments():
                        help="The wavelength range for the plot",
                        default=[None, None], type=float)
 
+    parser.add_argument("--exclude_bandnames", nargs='*',
+                        default=[],
+                        required=False,
```

Comment:
`required` is not necessary here. The default is `False`. Actually `type=str` is also not necessary as `str` is the default type.